doc/index.rst

```diff
@@ -1,78 +1,87 @@
+    :align: center
+    :width: 70%
+    :target: javascript:void(0);
 
-This PennyLane plugin allows the Strawberry Fields simulators to be used as PennyLane devices.
+|
 
-`Strawberry Fields <https://strawberryfields.readthedocs.io>`_ is a full-stack Python library for designing, simulating, and optimizing continuous variable (CV) quantum optical circuits.
+.. include:: ../README.rst
+  :start-after:	header-start-inclusion-marker-do-not-remove
+  :end-before: header-end-inclusion-marker-do-not-remove
 
-`PennyLane <https://pennylane.readthedocs.io>`_ is a machine learning library for optimization and automatic differentiation of hybrid quantum-classical computations.
 
+Once the PennyLane-SF plugin is installed, the two provided Strawberry Fields devices can be accessed
+straight away in PennyLane, without the need to import any additional packages.
 
-Features
-========
-
-* Provides two devices to be used with PennyLane: ``strawberryfields.fock`` and ``strawberryfields.gaussian``. These provide access to the Strawberry Fields Fock and Gaussian backends respectively.
-
+Devices
+=======
 
-* Supports all core PennyLane operations and observables across the two devices.
+PennyLane-SF provides two Strawberry Fields devices for PennyLane:
 
+.. devicegalleryitem::
+    :name: 'strawberryfields.fock'
+    :description: Full simulator that supports all continuous-variable operations.
+    :link: devices/fock.html
 
-* Combine Strawberry Fields' advanced quantum simulator suite with PennyLane's automatic differentiation and optimization.
+.. devicegalleryitem::
+    :name: 'strawberryfields.gaussian'
+    :description: Optimized simulator that supports only Gaussian operations.
+    :link: devices/gaussian.html
 
+.. raw:: html
 
-To get started with the PennyLane Strawberry Fields plugin, follow the :ref:`installation steps <installation>`, then see the :ref:`usage <usage>` page.
+        <div style='clear:both'></div>
+        </br>
 
+.. note::
 
-Authors
-=======
+    The Strawberry Fields plugin only supports
+    `continuous-variable (CV) operations <https://pennylane.readthedocs.io/en/stable/introduction/operations.html>`_,
+    such as ``Squeezing``, or ``NumberOperator``.
 
-Josh Izaac, Ville Bergholm, Maria Schuld, Nathan Killoran and Christian Gogolin
 
-If you are doing research using PennyLane and StrawberryFields, please cite our papers:
+Tutorials
+=========
 
-    Ville Bergholm, Josh Izaac, Maria Schuld, Christian Gogolin, and Nathan Killoran.
-    *PennyLane: Automatic differentiation of hybrid quantum-classical computations.* 2018.
-    `arXiv:1811.04968 <https://arxiv.org/abs/1811.04968>`_
+To see the PennyLane-SF plugin in action, you can use any of the *continuous-variable* based `demos
+from the PennyLane documentation <https://pennylane.ai/qml/demonstrations.html>`_, for example
+the tutorial on `Gaussian transformations <https://pennylane.ai/qml/demos/tutorial_gaussian_transformation.html>`_,
```

Comment:
Worth linking to the QNN tutorial as well? https://pennylane.ai/qml/demos/quantum_neural_net.html

Reply:
Good idea